modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -6,10 +6,19 @@
 [id="images-other-jenkins-config-kubernetes_{context}"]
 = Configuring the Jenkins Kubernetes plug-in
 
-The {product-title} Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] that allows Jenkins agents to be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
+The OpenShift Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] so that Jenkins agents can be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
```

Comment:
Should `Openshift Jenkins` be added as an attribute?

Reply:
Not in this case because it is only an image name, and might change.